Sources/Testing/Testing.docc/EnablingAndDisabling.md

```diff
@@ -101,7 +101,7 @@ If a test has multiple conditions applied to it, they must _all_ pass for it to
 run. Otherwise, the first condition to fail will be noted as the reason the test
 was skipped.
```

Comment:
```suggestion
run. Otherwise, the first condition to fail is noted as the reason why the test
was skipped.
```

Reply:
Well it hasn't happened yet—future tense to be avoided regardless?